code/game/objects/items/weapons/defib.dm

```diff
@@ -357,7 +357,7 @@
 
 				QDEL_NULL(ghost)
 			var/tplus = world.time - H.timeofdeath
-			var/tlimit = 3000 //past this much time the patient is unrecoverable (in deciseconds)
+			var/tlimit = istype(defib, /obj/item/defibrillator/bluespace) ? 5000 : 3000 //past this much time the patient is unrecoverable (in deciseconds)
```

Comment:
This is awful.

If stuff like this is going to be something modified, make it more OOP on the object itself, and not snowflake typechecks.

These kind of checks will not be tolerated.

Reply:
I didn't want to overrride the function since I'd have to paste that massive brick and only modify those small changes, not to mention I saw it in other places in the code so I thought it was ok.

Reply:
Yes, that's what I was implying.